packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -250,38 +313,45 @@ public class TableColumnPositiveNegativeNumber extends TableColumn<TableColumnPo
         return [valueKey];
     }
 
-    public readonly cellStyles: ElementStyles =
-        css`
-            .good {
-                color: green;
-            }
-
-            .bad {
-                color: red;
-            }
-        `;
-
-    public readonly cellTemplate: ViewTemplate<TableCellState<TableColumnPositiveNegativeNumberCellData>> =
-        html<TableCellState<TableColumnPositiveNegativeNumberCellData>`
-            <nimble-text-field
-                class="${x => isPositive(x.data.value) ? "good" : "bad"}"
-                readonly="true"
-                value="${x => x.data.value}"
-            >
-            </nimble-text-field>
-        `;
-
     public validateCellData(cellData: TCellRecord): void {
         if (typeof(cellData['value']) !== 'number') {
             throw new Error('Type for cellData is incorrect!');
         }
     }
 }
+
+class PositiveNegativeNumberCellElement extends BaseCellElement<TableColumnPositiveNegativeNumberCellData> {
+    @volatile
```

Comment:
nit: It doesn't look like the `textFieldCssClass` getter needs to be marked as `@volatile`. From my understanding, `@volatile` is needed when branching logic changes which observable properties are accessed, which is not the case in this example.

Reply:
Updated to remove `@volatile` from this example and the one above it. As you said, only the 1st example should need it as it has a branch between using `value` and `placeholder`.